stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -26,6 +26,25 @@ extension String {
   }
 }
 
+@_effects(readonly)
+private func _getClass(_ obj: AnyObject) -> AnyClass {
+  return object_getClass(obj)!
+}
+
+@_effects(readonly)
+private func _length(_ obj: AnyObject) -> Int {
+  return CFStringGetLength(unsafeBitCast(obj, to: CFString.self))
+}
+
+@_effects(releasenone)
+private func _copyString(_ obj: AnyObject) -> AnyObject {
+  return CFStringCreateCopy(kCFAllocatorSystemDefault, unsafeBitCast(obj, to: CFString.self))
+}
+
+private let (nscfClass, nscfConstantClass): (AnyClass, AnyClass) =
+  (objc_lookUpClass("__NSCFString")!,
+   objc_lookUpClass("__NSCFConstantString")!)
```

Comment:
Perhaps we should be investigating a way to have this provided via a function call instead of hard coding private class names into the overlay.

Reply:
Itai was under the impression we already had this hardcoded elsewhere, but I just looked and it seems we don't. I agree that would be better.

One idea I had here was introducing an "_isSafeToEagerBridge" method, though that runs into the usual issues with NSProxy like -isNSFoo__

Reply:
That may have been someone else (or I may have been unclear) — I also prefer exposing either a method or a function to look this up rather than hard coding.

Reply:
And such a method could be more general, e.g., true for more subclasses if there's no associated objects.